nips2018/deep_learning/README.md

```diff
@@ -0,0 +1,14 @@
+This example illustrate training of mnist on an hypersphere manifold.
```

Comment:
illustrates* the* training on a hypersphere. (?)

Reply:
change the name to deep_learning_hypersphere.py ? there are other examples of deep learning in the paper (eg SE(3)?)